jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -536,6 +516,29 @@ async def put(self, path):
             raise tornado.web.HTTPError(status_code=500, reason=repr(e))
         await self.finish()
 
+class MountPfsRepoHandler(BaseHandler):
+    @tornado.web.authenticated
+    async def put(self):
+        try:
+            body = self.get_json_body()
+            if not body:
+                raise ValueError("branch_uri does not exist")
+            branch_uri = body.get("branch_uri")
+            if branch_uri is None:
+                raise ValueError("branch_uri does not exist")
+            if branch_uri == '':
```

Comment:
is this an unmount mechanism?

Reply:
Yep. Alternatively maybe we go back to the separate unmount endpoint? In hindsight that sounds better.

Reply:
Ok reverted this to be a separate unmount endpoint